### ISSUE TITLE
babel-runtime is outdated, can't be updated (and, somehow, exists 61 times)

### ISSUE THREAD

**Issue by MarkTiedemann** [babel/babel#2855]
So to test `babel`, I ran a bunch of commands:

```
mkdir babel-test
cd babel-test
echo "export async function foo(){}" > src.js
echo "{ \"presets\": [ \"es2015\", \"stage-0\" ] }" > .babelrc
npm i babel-cli@latest -g
npm i babel-preset-es2015@latest
npm i babel-preset-stage-0@latest
```

When I ran `babel src.js` now, I got the following error:

```
Error: src.js: Expected type "Identifier" with option {}
    at Object.t.(anonymous function) [as assertIdentifier] (C:\Sandbox\babel-test\node_modules\babel-types\lib\index.js:80:13)
    at PluginPass.exports.visitor.Function.exit (C:\Sandbox\babel-test\node_modules\babel-plugin-transform-regenerator\lib\visit.js:78:9)
    at c:\Program Files\nodejs\node_modules\babel-cli\node_modules\babel-traverse\lib\visitors.js:271:19
    at NodePath._call (c:\Program Files\nodejs\node_modules\babel-cli\node_modules\babel-traverse\lib\path\context.js:72:18)
    at NodePath.call (c:\Program Files\nodejs\node_modules\babel-cli\node_modules\babel-traverse\lib\path\context.js:44:17)
    at NodePath.visit (c:\Program Files\nodejs\node_modules\babel-cli\node_modules\babel-traverse\lib\path\context.js:108:8)
    at TraversalContext.visitQueue (c:\Program Files\nodejs\node_modules\babel-cli\node_modules\babel-traverse\lib\context.js:151:16)
    at TraversalContext.visitMultiple (c:\Program Files\nodejs\node_modules\babel-cli\node_modules\babel-traverse\lib\context.js:106:17)
    at TraversalContext.visit (c:\Program Files\nodejs\node_modules\babel-cli\node_modules\babel-traverse\lib\context.js:193:19)
    at Function.traverse.node (c:\Program Files\nodejs\node_modules\babel-cli\node_modules\babel-traverse\lib\index.js:139:17)
```

Since I wasn't sure what had caused the error, I ran `babel-doctor`:

```
Babel Doctor
Running sanity checks on your system. This may take a few minutes...

√ Found config at C:\Sandbox\babel-test\.babelrc
√ No duplicate babel packages found

× We found some outdated packages:

- babel-runtime - Latest is 6.0.14. Local version is 5.8.29

√ You're on npm >=3.3.0

Found potential issues on your machine :(
```

To fix this, I ran `npm i babel-runtime@latest`. However, running `babel-doctor` again produced this output:

```
Babel Doctor
Running sanity checks on your system. This may take a few minutes...

√ Found config at c:\Sandbox\babel-test\.babelrc

× Found these duplicate packages:

- babel-runtime x 61

Recommend running `npm dedupe`

× We found some outdated packages:

- babel-runtime - Latest is 6.0.14. Local version is 5.8.29
/* REMOVED 59 LINES HERE */
- babel-runtime - Latest is 6.0.14. Local version is 5.8.29

√ You're on npm >=3.3.0

Found potential issues on your machine :(
```

So I ran `npm dedupe babel-runtime` and `babel-doctor` again, but nothing changed.

I figured, perhaps I had to update the global module. Before doing this, I first reset the local modules by running `rm -rf node_modules`, `npm i babel-preset-es2015@latest` and `npm i babel-preset-stage-0@latest`. I then ran `npm i babel-runtime -g` and `babel-doctor` again, but it still said that `babel-runtime` was outdated.

Now, I don't know what else to do to fix this. Any help is much appreciated. :)

**Comment by kzc** [comment]
Delete all babel related node_modules installed globally and locally and then use npm 3.3.6 or greater to install babel 6. You have to start from a clean slate to get it to work. Any previous trace of babel 5 modules or npm 2.x installed modules can negatively impact babel 6 in my experience.

**Comment by MarkTiedemann** [comment]
I removed `.babel.json` from `~/ `, `npm` and `npm-cache` from `~/AppData/Roaming/` and all globally installed npm and babel modules from `/c/Program Files/nodejs/`.

I also, for no reason, reinstalled Node.js, then reinstalled npm (which got upgraded form 3.3.0 to 3.3.6) and, finally, on the fresh system, reinstalled babel-cli -- and it still doesn't work.

```
Babel Doctor
Running sanity checks on your system. This may take a few minutes...

√ Found config at c:\Sandbox\babel-test\.babelrc
√ No duplicate babel packages found

× We found some outdated packages:

- babel-runtime - Latest is 6.0.14. Local version is 5.8.29

√ You're on npm >=3.3.0

Found potential issues on your machine :(
```

I checked the `package.json` from the new globally installed babel-cli. Turns out it only has `"babel-runtime": "^5.0.0"` as its dependency.

So a freshly installed babel-cli has an oudated babel-runtime so babel-doctor says there are issues on my machine? I don't get it. :S

Still not fixed.

**Comment by hzoo** [comment]
You're initial issue (`Expected type "Identifier" with option {}`) with `export async function foo(){}` looks to be the same as https://github.com/babel/babel/issues/2835 which is probably a bug so we need to look into that.

You don't need the runtime to be 6.0 so I wouldn't worry about that

**Comment by MarkTiedemann** [comment]
@hzoo Thanks for looking into it!

If you don't need babel-runtime 6.0, shouldn't babel-doctor stop marking it as an issue? That kinda confused me at least.

**Comment by hzoo** [comment]
Yeah - it's new so might have some issues. Going to close for #2835. (Might have a fix soon?)

> For questions and support please visit the <a href="https://babel-slack.herokuapp.com">Slack community</a> or <a href="http://stackoverflow.com/questions/tagged/babeljs">StackOverflow</a>. The Babel issue tracker is <strong>exclusively</strong> for bug reports and feature requests.

**Comment by MarkTiedemann** [comment]
The babel-doctor warnings seem to be fixed as mentioned here #2915.
